src/models/settlement/facade.js

```diff
@@ -758,7 +761,7 @@ const Facade = {
         const settlementData = await knex('settlement AS s')
           .join('settlementStateChange AS ssc', 'ssc.settlementStateChangeId', 's.currentStateChangeId')
           .join('settlementModel AS sm', 'sm.settlementModelId', 's.settlementModelId')
-          .select('s.settlementId', 'ssc.settlementStateId', 'ssc.reason', 'ssc.createdDate', 'sm.autoPositionReset')
+          .select('s.settlementId', 'ssc.settlementStateId', 'ssc.reason', 'ssc.createdDate', 'sm.autoPositionReset', 'sm.requireLiquidityCheck')
```

Comment:
Need to do. a select for updated to lock the participant positions before updating